coincube-ui/Cargo.toml

```diff
@@ -5,21 +5,25 @@ edition = "2021"
 description = "COINCUBE UI components - Reusable Iced widgets for COINCUBE wallet"
 repository = "https://github.com/coincubetech/coincube"
 
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+[features]
+debug = ["iced/hot"]
```

Comment:
Remove this until we get it working in some capacity, otherwise we'll be pulling in a whole slew of unused dependencies